### ISSUE TITLE
[App]: When using "Feature/" , it doesn't display the images

### ISSUE THREAD

**Issue by bpetit2019** [wandb/wandb#4231]
### Current Behavior

When I log images in wandb with:
wandb.log({"Feature/something":wandb.Image(...)}) it displays points and not the images with steps. The images are in the files though.
When I log with:
wandb.log({"Features/something":wandb.Image(...)}) or something else than "Feature", it works correctly and displays images

### Expected Behavior

It should display the images with steps.

### Screenshots

![image](https://user-images.githubusercontent.com/95279738/188670185-3640cad9-07af-47b1-9c67-eadce768b499.png)

### Environment

OS:

Browsers:

Version:wandb 0.12.18

**Comment by bpetit2019** [comment]
Expected behavior
![image](https://user-images.githubusercontent.com/95279738/188670603-fecb92b6-3291-44ce-a450-134c963a8c5c.png)

**Comment by MBakirWB** [comment]
Hi @bpetit2019, I tested logging images with incremental steps on W&B 0.12.8 and greater and did not experience similar behavior. See this example [project](https://wandb.ai/mohammadbakir/logging_image_key_test) where I logged images using various combinations of keys. Additionally select each of the runs and visit their saved code page, example of [one](https://wandb.ai/mohammadbakir/logging_image_key_test/runs/3uxpbr34/code?workspace=user-mohammadbakir). Verify that you are logging your images correctly. If you are unable to produce the expected results,  share a snippet of your image logging code and we'll take a closer look.

**Comment by bpetit2019** [comment]
Maybe this can help you
```wandb.login()
wandb.init(project="project", sync_tensorboard=True)
wandb.log({"Feature/p2": wandb.Image(turn_into_img(features["p2"].cpu().detach().numpy()[0, 0])))```
where `features["p2"]` is a torch tensor and the function turn_into_img returns an array of integers between 0 and 255

**Comment by MBakirWB** [comment]
Hi @bpetit2019 , we've been unable to replicate your specific issue. We can reattempt if you could provide an expanded version of your code example. What is the dim size of the your tensor? What are you specifically doing in the function turn_into_img? wandb.Image will accept a 1-D array but will display a warning in the charts panel in the workspace that reads, "Selected runs are not logging media for the key <key>, but instead are logging values of type histogram (or list)."

**Comment by MBakirWB** [comment]
Hi @bpetit2019 , since we have not heard back from you we are going to close this request. If you would like to re-open the conversation, please let us know!
